facilities/models/rating_models.py

```diff
@@ -0,0 +1,200 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+
+from common.models import AbstractBase
+
+from facilities.models import Facility, FacilityService
+from users.models import MflUser
+
+
+class RatingScale(AbstractBase):
+    """
+    Holds the different types of rating of an entity.
+
+    Some organisations use the likert scale, others use the interval
+    scale and yet others use the ration scale.
+    """
+
+    name = models.CharField(
+        max_length=100, unique=True,
+        help_text='The name of the rating scale e.g likert RatingScale')
+    description = models.TextField(
+        help_text='A summary of the type of rating scale. E.g What it is. ')
+
+    def __unicode__(self):
+        return self.name
+
+
+class Rating(AbstractBase):
+    """
+    The different levels of a scale.
+
+    The format of a typical five-level Likert item, for example, could be:
+
+        1. Strongly disagree
+        2. Disagree
+        3. Neither agree nor disagree
+        4. Agree
+        5. Strongly agree
+    """
+
+    scale = models.ForeignKey(RatingScale, on_delete=models.PROTECT)
+    rating_code = models.CharField(
+        max_length=30,
+        help_text='A code representing  a rate e.g 1, Likely etc')
+    description = models.TextField(
+        null=True, blank=True,
+        help_text='An explanation of how the rating code is used')
+
+    def __unicode__(self):
+        return self.scale.name + ": " + self.rating_code
+
+    class Meta:
+        unique_together = ('scale', 'rating_code', )
+
+
+class FacilityRatingScale(AbstractBase):
+    """
+    The rating scale that a facility uses.
+    """
+
+    facility = models.ForeignKey(
+        Facility, on_delete=models.PROTECT,
+        related_name='facility_rating_scales')
+    scale = models.ForeignKey(
+        RatingScale, on_delete=models.PROTECT,
+        related_name='facilities_using_scale')
+
+    def __unicode__(self):
+        return self.facility.name + ": " + self.scale.name
+
+    def validate_only_one_active(self):
+        facility_scales_no = self.__class__.objects.filter(
+            facility=self.facility, scale=self.scale, active=True).count()
+        if facility_scales_no > 0:
+            raise ValidationError(
+                "Only one type scale can be active for a facility at a time")
+
+    def save(self, *args, **kwargs):
```

Comment:
`AbstractBase` already calls `full_clean`; you implement a custom `clean`, you will be OK.